fix(a11y): final fix for WCAG 1.1.1 image alt should describe image content accurately (#17)
scottrigby/basekamp-next: content/projects/walk-talk-eat-talksomemore.md

attribution: Co-organized by Basekamp & c.cred
images:
  - src: basekamp1_0.jpg
    alt: Group of participants sitting on plastic milk crates in gallery space during discussion, with projection screen showing map
  - src: basekamp2_0.jpg
    alt: Participants walking along urban sidewalk carrying bright blue flag markers during counter-cartography walk
  - src: meal01.jpg

>     alt: Six people seated around table with blue tablecloth having meal and discussion, food bowls visible

Reviewer: ```suggestion
    alt: Five people seated around table with blue tablecloth having meal and discussion, food bowls visible
```

  - src: webcast02.jpg
    alt: Grid of video conference feeds showing participants from multiple cities including Philadelphia, Portland, New York, Chicago, Malmö, London, Copenhagen, and Atlanta
  - src: 01_1.jpg
    alt: Two men standing beside "Welcome to Cabbagetown" sign and yard sale sign on grass lawn
  - src: 03_0.jpg
    alt: Graffitied concrete railroad overpass bridge with "KISS ATLANTA" text and car passing underneath
  - src: 01_2.jpg
    alt: White tent canopy set up in vacant urban lot with food and supplies on table, apartment buildings visible in background
  - src: 06_0.jpg
    alt: Triangular green pedestrian crossing sign heavily covered with stickers, graffiti tags, and street art
  - src: 1_1.jpg

>     alt: Street map with hand-drawn black marker lines tracing walking route through urban neighborhoods

Reviewer: ```suggestion
    alt: Street map with hand-drawn black marker lines tracing walking route
```

  - src: meal1_0.jpg
    alt: Empty kitchen and dining area with wooden floors, simple table and chairs, domestic interior
  - src: meal2_0.jpg
    alt: Group of people seated on couch and chairs watching projected image in living room during webcast event
  - src: 08_0.jpg
    alt: White railing or banner with hand-painted red text reading "LOOK ↑ TREEROOT"
  - src: 02_0.jpg
    alt: View beneath bridge showing water, concrete pillars, and debris-strewn muddy bank
  - src: Jessica.jpg
    alt: Woman in burgundy leather jacket with yellow patterned scarf holding equipment at night during outdoor event
  - src: Guests.jpg
    alt: Nighttime outdoor gathering with arrangement of flowers in green glass bottles, lit candles, and drink can on surface
  - src: tampa-crossing.jpg
    alt: Group of five people standing at wide pedestrian crosswalk with traffic lights and urban buildings in background
  - src: tampa-highway3.jpg
    alt: Line of people walking along highway shoulder beside busy multi-lane road with overpass visible ahead
  - src: meal1_1.jpg
    alt: Nighttime outdoor gathering with people sitting in circle on pavement around illuminated center point
  - src: meal2_1.jpg

>     alt: White dome-shaped outdoor pizza oven with fire burning inside wooden structure at night

Reviewer: ```suggestion
    alt: White dome-shaped outdoor pizza oven with fire burning inside and wood piled in front of the opening at night
```
